tofhir-server/src/main/scala/io/tofhir/server/service/schema/SchemaFolderRepository.scala

```diff
@@ -354,5 +334,92 @@ class SchemaFolderRepository(schemaRepositoryFolderPath: String, projectFolderRe
     }
     baseFhirConfig
   }
+
+  /**
+   * @param projectId Project containing the schema definition
+   * @param id        Identifier of the schema definition
+   * @return          Structure definition of the schema
+   */
+  override def getSchemaAsStructureDefinition(projectId: String, id: String): Future[Option[Resource]] = {
+    getSchema(projectId, id).map {
+      case Some(schemaStructureDefinition) =>
+        Some(SchemaUtil.convertToStructureDefinitionResource(schemaStructureDefinition))
+      case None =>
+        None
+    }
+  }
+
+  /**
+   * Save a schema by using directly its structure definition resource
+   * @param projectId
```

Comment:
Complete the JavaDoc and describe the exceptions that can be thrown in the function.